File: Herencia/ejercicio 3/Java/Persona.java
import java.time.LocalDate;
import java.time.Period;
import java.util.*;
class Persona {
    public String ci, nombre, apellido, celular, sexo;
    public LocalDate fechaNac;
    public Persona(String ci, String nombre, String apellido, String celular, LocalDate fechaNac, String sexo) {
        this.ci = ci;
        this.nombre = nombre;
        this.apellido = apellido;
        this.celular = celular;
        this.fechaNac = fechaNac;
        this.sexo = sexo;
    }
    public int getEdad() {
        return Period.between(fechaNac, LocalDate.now()).getYears();
    }
    public void mostrar() {
        System.out.println(nombre + " " + apellido + ", CI: " + ci + ", Edad: " + getEdad());
    }
    public String getApellido() {
        return apellido;
    }
    public String getSexo() {
        return sexo;
    }
}

File: Herencia/ejercicio 3/Python/Persona.py
from datetime import date
class Persona:
    def __init__(self, ci, nombre, apellido, celular, nacimiento, sexo):
        self.ci = ci
        self.nombre = nombre
        self.apellido = apellido
        self.celular = celular
        self.nacimiento = nacimiento
        self.sexo = sexo
    def edad(self):
        h = date.today()
        return h.year - self.nacimiento[0] - ((h.month, h.day) < (self.nacimiento[1], self.nacimiento[2]))
    def mostrar(self):
        print(f"{self.nombre} {self.apellido}, CI: {self.ci}, Edad: {self.edad()}")